src/pages/networks/[blockchain]/[network]/blocks.tsx

```diff
@@ -35,7 +35,7 @@ const BlocksPageRoute = ({
     if (!blockchain || !network) {
         return <span />;
     }
-    if (!blocks || !maxBlock || !currentBlock) {
+    if (!blocks) {
```

Comment:
when there is only a genesis block, maxBlock == 0 and currentBlock == 0. those are perfectly fine values